### PR TITLE
navigation a11y

### PR DESCRIPTION
**Pull request by yeion7**
## What kind of change does this PR introduce?

Improve

## What is the current behavior?

navigation has a few a11y issues

## What is the new behavior?

d8RTvWPZwM

## Checklist

- [ ] Documentation
- [ ] Testing 
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

**Comment by yeion7** [comment]
not sure about the `typecheck` error, it sounds like cliche but in my local works lol

image

**Comment by SaraVieira** [comment]
The hero we needed

File: packages/app/src/app/pages/common/Navigation/index.tsx [hunk 1]
```diff
@@ -105,8 +109,14 @@ export const Navigation = inject('store', 'signals')(
...
                     {({ store }) => (
                       <Action
+                        as="button"
                         style={{ position: 'relative', fontSize: '1.25rem' }}
                         onClick={open}
+                        aria-label={
+                          store.userNotifications.unreadCount > 0
+                            ? 'Show Notifications'
+                            : 'No Notifications'
+                        }
```
Comment (SaraVieira) [inline packages/app/src/app/pages/common/Navigation/index.tsx]: 💯

**Comment by SaraVieira** [comment]
THANK YOU!

This is amazing! Our hero!

**Comment by yeion7** [comment]
@SaraVieira thanks for the review in these PRs, I've learned a lot with the sandbox codebase 😁if you see something to improve, let me know

**Comment by jyash97** [comment]
I just checked this by pulling the changes. This is really awesome work 💯 @yeion7

**Pull request merged**